datahub-web-react/src/app/identity/user/UserListItem.tsx

```diff
@@ -103,6 +157,24 @@ export default function UserListItem({ user, canManageUserCredentials, onDelete
                 </Link>
             </UserItemContainer>
             <ButtonGroup>
+                <RoleSelect
+                    placeholder={
+                        <RoleIcon>
+                            <UserOutlined />
+                            No Role
+                        </RoleIcon>
+                    }
+                    value={userRoleUrn || undefined}
+                    onChange={(e) => {
```

Comment:
Usually we tend to extract these into const methods above.. onSelectRole method for example